frontend/packages/dev-console/features/pipeline-Add_pages_for_tekton_trigger

```diff
@@ -0,0 +1,89 @@
+Feature: Add page for Tekton Trigger
+	As a user, I want see a page for Tekton Triggers
+
+Background:
+    Given user logged into the openshift application
+    And Pipelines operator should be installed
+
+@regression, @smoke
+Scenario: Adding triggers from developer view
+   Given User is in pipeline section in developer view having a pipeline
+   When User click on kebab menu of the pipeline
+   And  User selects Add Trigger
+   Then Add Trigger dialog box opens
+   And  Add Git Provider Type as github-pullreq 
+   And  User provide git resource and image resources if required
+   And  User clcks on Add
```

Comment:
Then statement should not have any actions. Replace Then with When if you want to perform any actions.